pages/Timeline.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react'
+import { getAchievements, getMinAchievementDate, getHighlights } from '../lib/data'
+import { Head, FlatButton } from '../components/Core'
+import TimelineComponent from '../components/Timeline'
+import styles from '../styles/Timeline.module.css'
+
+interface TimelineProps {
+  achievements: AchievementData[],
+  minAchievementDate: MinAchievementDateData,
+  highlights: HighlightData[]
+}
+
+const Timeline: FC<TimelineProps> = ({ achievements, minAchievementDate, highlights }) => {
+  return (
+    <>
+      <Head
+        title='Timeline'
+        description='The story of my career by Johannes Palvanen.' />
+      <a href='/'>
+        <FlatButton className={styles.closeButton}>
+          X
+        </FlatButton>
+      </a>
+      <TimelineComponent
+        achievements={achievements}
+        minAchievementDate={minAchievementDate}
+        highlights={highlights} />
+    </>
+  )
+}
+
+export async function getServerSideProps() {
+  const achievements: AchievementData[] = await getAchievements()
```

Comment:
to minimise / parallelise loading, use Promise.all

Reply:
Typecasting is a bich, try to return correct type from getAchievements function itself